README.md

```diff
@@ -89,6 +96,13 @@ For Docker on Mac users, remember to add `tiab/volume/onos` to shared folder lis
     Double VLAN pop and route is no longer supported with OVS. We have moved forward to implementing that with P4 BNG. Therefore the follow one doesn't work either.
     - `trellis_double_tagged`
 
+- netcfg
+
+    The mininet container will push the `${TOPO}`.json to ONOS controller by default, and you can override this behavior by providing your config file.
+    - Place your file under `volume/mininet` and replace the environment variable `CFG_FILE` in `.env` file with your file name.
+    - Restart the mininet by the command `make restart_mininet` and then check the log by running `docker logs -f mininet | grep "Check custom config"`
```

Comment:
We should explain that the new configuration will be pushed as part of Mininet restart.